n3fit/src/n3fit/checks.py

```diff
@@ -393,3 +393,21 @@ def check_deprecated_options(fitting):
     for option in nnfit_options:
         if option in fitting:
             log.warning("'fitting::%s' is an nnfit-only key, it will be ignored", option)
+
+@make_argcheck
+def check_fiatlux_pdfs_id(replicas, fiatlux, replica_path):
+    if fiatlux :
+        import lhapdf
+        pdf_name = fiatlux["pdf_name"]
+        pdfs_fiatlux = lhapdf.getPDFSet(pdf_name)
+        max_id = max(replicas)
+        pdfs_ids = pdfs_fiatlux.size - 1
```

Comment:
```suggestion
        from validphys.core import PDF
        pdf_name = fiatlux["pdf_name"]
        pdfs_ids = PDF(pdf_name).get_members() - 1 # get_members counts also replica0
        max_id = max(replicas)
```

Preferably keep using vp objects as an interface to lhapdf. I also think the import can be moved to the top of the module.